scenes/game_elements/characters/player/components/player_interaction.gd

```diff
@@ -28,7 +30,7 @@ func _process(_delta: float) -> void:
 		interact_zone.monitoring = false
 		interact_label.visible = false
 		interact_area.interaction_ended.connect(_on_interaction_ended, CONNECT_ONE_SHOT)
-		interact_area.start_interaction(interact_zone.is_looking_from_right)
+		interact_area.start_interaction(player, interact_zone.is_looking_from_right)
```

Comment:
For another moment, but if we are passing the player as a parameter, I think we won't need to pass which side the player's looking at as well.